vulnerability-report.md

```diff
@@ -0,0 +1,55 @@
+# A1 - Injection
+## How you found and exploited the vulnerability, including the code/method you used?
+- Uploading an image with sql in the file name allowed us to create an image that when clicked
+exposes the hashed username and password in the Django error message.  The exact sql is:
+    - testPic',(select password from auth_user where username='admin'),7);--
+
+## What you were able to do with the vulnerability (i.e. what was exposed?)
+- We were able to see the hashed password and username for the user with the error message, not that much else was revealed.
+
+## The code that fixes the vulnerability
+- Creating the file with defined values and saving the file would prevent sql injection, instead of directly executing the code to the database.
+
+
+# A2 - Broken Auth
+## How you found and exploited the vulnerability, including the code/method you used
+- On the user creation form we added checkboxes for is_admin, is_staff, is_superuser.
```

Comment:
How did you add these fields? Did you edit a file? Did you modify the HTML in browser?